xarray/tests/test_extensions.py

```diff
@@ -5,9 +5,14 @@
 import pytest
 
 import xarray as xr
+
+# TODO: Remove imports in favour of xr.DataTree etc, once part of public API
```

Comment:
I'm also hoping that once we can for `xr.DataTree` that all the type annotations that `mypy` has insisted on in the tests might be able to be removed (e.g., `dt: DataTree = DataTree()`)